.gitlab-ci.yml

```diff
@@ -2,12 +2,46 @@ default:
   image: 'docker.repo.splunkdev.net/ci-cd/ci-container:debian-buster'
 
 stages:
+  - sast-oss-scan
   - build
   - sign
   - package
   - release
   - github-release
 
+include:
+  # https://cd.splunkdev.com/prodsec/scp-scanning/gitlab-checkmarx
+  - project: 'prodsec/scp-scanning/gitlab-checkmarx'
+    ref: latest
+    file: '/templates/.sast_scan.yml'
+  # https://cd.splunkdev.com/prodsec/whitesource
+  - project: 'prodsec/whitesource'
+    ref: latest
+    file: '/templates/.whitesource.yml'
+
+semgrep:
+  stage: sast-oss-scan
+  extends: .sast_scan
+  variables:
+    SAST_SCANNER: "Semgrep"
+    SEMGREP_EXCLUDE: "examples,internal/buildscripts,tests,*_test.go"
+    alert_mode: "policy"
+  after_script:
+    - echo "Check results at $CI_PIPELINE_URL/security"
+  only:
+    - main
+
+whitesource:
```

Comment:
If these fail is it easy to tell what the problem is?  I can't seem to access the report w/ sso.